### ISSUE TITLE
About logical router policies route configured after  stattic route on the logical router

### ISSUE THREAD

**Issue by ts170710** [ovn-org/ovn#119]
In OVN logical flows table, the table number of Router policies table (lr_in_policy) in ovn logical router is 13, and IP Routing table(lr_in_ip_routing) is 11.
The priority of router policy route is higher than that of static route in linux kernel.
Learn it that PBR can override routing decisions and provide a different next-hop from [1].
We encountered such a scene,  after deleting the default route of static route, the flow table in table 11(lr_in_ip_routing) can not match and advances to the next table. Then the PBR cannot take effect.
In other words, PBR depends on lr_in_ip_routing table match and advances to next table.
Is this a bug?

[1] https://patchwork.ozlabs.org/project/openvswitch/patch/1554334195-126528-2-git-send-email-mary.manohar@nutanix.com/

**Comment by almusil** [comment]
Hi,

this issue has been inactive for more than 6 months, closing it as stale. In case the issue still persists with recent version of OVN please feel free to re-open it with info from new version.

Thanks,
Ales
